server.js

```diff
@@ -4,31 +4,72 @@ server.js
 starts server on port 3000
 */
 
-var express = require('express');
-var bodyParser = require('body-parser');
+const express = require('express');
+const bodyParser = require('body-parser');
+const swaggerJSDoc = require('swagger-jsdoc');
+const swaggerUi = require('swagger-ui-express');
 
-var index = require('./routes/index');
-var pinocchio = require('./routes/pinocchio');
-var api = require('./routes/api');
+// load app config
 const { reactServer } = require('./config.js');
 
+// create express app
 var app = express();
-var port = process.env.PORT || 3000;
+var portNumber = 3000;
+var port = process.env.PORT || portNumber;
 
+// swagger definition
+const swaggerDefinition = {
+  info: {
+    title: 'Pinocchio node backend server',
+    version: '1.0.0',
+    description: 'API endpoints to handle Pinocchio system routes',
+  },
+  host: 'localhost:'+3010,//portNumber,
+  basePath: '/',
+  securityDefinitions: {
+    token: {
+      type: 'apiKey',
+      name: 'token',
+      scheme: 'bearer',
+      in: 'header',
+    },
+  },
+};
+
+// swagger options
+const options = {
+  swaggerDefinition,
+  apis: ['./src/routes/*.js'],
+};
+
+const swaggerSpec = swaggerJSDoc(options);
+
+app.get('/swagger.json', (req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+  res.send(swaggerSpec);
+});
```

Comment:
for swagger configuration, you can implement it in another js file (?) Just to enhance readability.